server/controllers/accommodationController.js

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable max-len */
+import accommodationHelper from '../helpers/accommodationHelper';
+import picture from '../helpers/uploadImage';
+
+/**
+ * This class contains all methods
+ * required to handle
+ * accommodations' request.
+ */
+class AccommodationController {
+  /**
+   * This method handle the accommodation request.
+   * @param {object} req The accommodation's request.
+   * @param {object} res The response.
+   * @returns {object} The status and some data of the accomodation.
+   */
+  static async supply(req, res) {
+    try {
+      if (req.files && req.files.image) {
+        console.log('hello');
+        let image;
+        if (req.files.image.type || req.files.image.length) {
+          image = await picture.uploader(req.files.image);
+        } else {
+          return res.status(400).json({ status: 400, error: 'Please select one or more pictures' });
+        }
+        if (!image || image.includes('null')) {
+          return res.status(415).json({ status: 415, error: 'Please select the right type of image' });
+        }
```

Comment:
Try to refactor these codes, too many if statements.

Reply:
alright